package/firmware/ipq-wifi/Makefile

```diff
@@ -28,6 +28,7 @@ ALLWIFIBOARDS:= \
 	8dev_habanero-dvk \
 	aruba_ap-303 \
 	avm_fritzrepeater-1200 \
+	buffalo_wtr-m2133hp \
```

Comment:
Can you please send both boardfiles upstream to ath10k ML?

The Process is explained on the  [linux-wireless wiki ](https://wireless.wiki.kernel.org/en/users/drivers/ath10k/boardfiles) [There's also a comment in this Makefile about it!]. You can use previous submissions as a template. 

ipq40xx:
https://www.mail-archive.com/ath10k@lists.infradead.org/msg10923.html

qca9984:
http://lists.infradead.org/pipermail/ath10k/2018-September/011990.html

Reply:
Okay. I sent all BDFs to the ath10k mailing list.
https://lists.infradead.org/pipermail/ath10k/2020-April/014873.html
https://lists.infradead.org/pipermail/ath10k/2020-April/014874.html